xdocs/usermanual/jmeter_tutorial.xml

```diff
@@ -110,12 +110,28 @@ The folders inside of <code>src</code>
 </p>
 
 <dl>
+<!--
+  <dt><code>bom</code></dt><dd></dd>
+-->
+  <dt><code>bshclient</code></dt><dd>code for the BeanShell based client</dd>
   <dt><code>components</code></dt><dd>contains non-protocol-specific components like visualizers, assertions, etc.</dd>
+<!--
+  <dt><code>config</code></dt><dd>XXX</dd>
+-->
   <dt><code>core</code></dt><dd>the core code of JMeter including all core interfaces and abstract classes.</dd>
+  <dt><code>dist</code></dt><dd>builds script that creates a distribution</dd>
+  <dt><code>dist-check</code></dt><dd>code related to testing the distribution. It is the place to look for
+    when you want to update the contents of the resulting source/binary archive</dd>
   <dt><code>examples</code></dt><dd>example sampler demonstrating how to use the new bean framework</dd>
   <dt><code>functions</code></dt><dd>standard functions used by all components</dd>
+  <dt><code>generator</code></dt><dd>code to generate a test plan with all elements. Used for testing the distribution</dd>
   <dt><code>jorphan</code></dt><dd>utility classes providing common utility functions</dd>
+  <dt><code>launcher</code></dt><dd>code to help start and stop JMeter through API</dd>
+  <dt><code>licenses</code></dt><dd>contains information about the licenses used in JMeters dependencies</dd>
   <dt><code>protocol</code></dt><dd>contains the different protocols JMeter supports</dd>
+  <dt><code>release</code></dt><dd>code related to releasing JMeter distribution</dd>
+  <dt><code>testkit</code></dt><dd>utility code for testing</dd>
+  <dt><code>testkit-wiremock</code></dt><dd>utility code for testing with WireMock</dd>
```

Comment:
With addition of bolt, this should be updated

Reply:
I wonder if we can "autogenerate" this section.
Otherwise it will always be out of date.

Of course, I don't suggest to implement "autogeneration" in this PR. I just suggest we don't block the merge in case "oh, `bolt` or `bom` module is not mentioned`.